File: common/src/main/java/net/impleri/playerskills/integration/kubejs/PlayerSkillsKubeJSWrapper.java
package net.impleri.playerskills.integration.kubejs;

import net.impleri.playerskills.api.Skill;
import net.impleri.playerskills.api.SkillType;

import java.util.List;

class PlayerSkillsKubeJSWrapper {
    public List<Skill<?>> getSkills() {
        return net.impleri.playerskills.server.api.Skill.all();
    }

    public List<SkillType<?>> getSkillTypes() {
        return SkillType.all();
    }

    public List<SkillType<?>> getTypes() {
        return getSkillTypes();
    }
}

File: common/src/main/kotlin/net/impleri/playerskills/integrations/kubejs/skills/PlayerSkillsKubeJSWrapper.kt
package net.impleri.playerskills.integrations.kubejs.skills

import net.impleri.playerskills.api.Skill
import net.impleri.playerskills.api.SkillType

internal class PlayerSkillsKubeJSWrapper {
  fun getSkills(): List<Skill<*>> {
    return Skill.all()
  }

  fun getSkillTypes(): List<SkillType<*>> {
    return SkillType.all()
  }

  fun getTypes(): List<SkillType<*>> {
    return getSkillTypes()
  }
}
